aptos-move/framework/aptos-stdlib/sources/table_with_length.spec.move

```diff
@@ -0,0 +1,45 @@
+/// Specifications of the `Table` module.
```

Comment:
Don't need that. `intrinsic` means its build into the prover and it isn't because its based on the other table.